File: Examples/0/ProbA.vb
Option Explicit On

Module HelloWorld

	' The Main subroutine is the entry point for your program;
	' make sure to include it! This file can be used as a
	' template for solving all Code Quest problems
    Sub Main(args As String())
		' All inputs starts with the number of test cases.
		' These next two lines should appear in every solution.
        Dim i As Integer
        Int32.TryParse(Console.ReadLine(), i)
		' Now we need to do something for each test case. This
		' For loop allows us to do that. These two lines are the
		' last two common to all problems; inside the loop, it'll
		' be different for each problem.
        Dim test As Integer = 0
        For test = 0 To (i - 1)
			' For Hello World, we just want to print out whatever
			' text we read in.
            Console.WriteLine(Console.ReadLine())
        Next
    End Sub

End Module

File: Examples/0/ProbA.cpp
// Recommended includes for all problems. Some problems
// may require additional libraries.
#include <iostream>
#include <string>
#include <cmath>
#include <cstdlib>
using namespace std;

// main() serves as the entry point to your program - 
// this is how your program will be run. You can use this
// file as a template for solving other problems.
int main()
{
	// All inputs start with the number of test cases. These two
	// lines should always appear in every problem.
	int testCases;
	cin >> testCases;
	// This string and the following "getline" call move the 
	// input cursor past the newline character at the end of
	// the first line. This is important to read more input.
	string dummy;
	getline(cin, dummy);
	
	// This for loop allows us to do the same thing for each
	// test case. This line is the last one common to all
	// problems - inside this loop, things will be different.
	for(int testcase = 0; testcase < testCases; testcase++){
		// For Hello World, we simply want to output whatever
		// text we read in.
		string text;
		getline(cin, text);
		cout << text << '\n';
	}
}
